tests/integration/rules/callback.phpt

```diff
@@ -0,0 +1,37 @@
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Exceptions\CallbackException;
+use Respect\Validation\Exceptions\NestedValidationException;
+use Respect\Validation\Validator as v;
+
+try {
+    v::callback('is_string')->check([]);
+} catch (CallbackException $exception) {
+    echo $exception->getMessage().PHP_EOL;
+}
+
+try {
+    v::not(v::callback('is_string'))->check('foo');
```

Comment:
@henriquemoody this test don't covers `v::not(v::callback('is_string'))->check(...);` test?